### PR TITLE
Add scripts

### PR DESCRIPTION
**Pull request by bnika**
Added data preprocessing and training scripts.
Extended Readme accordingly.

### CODE REVIEW

File: sweep-is [hunk 1]
```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+# Sweeps over data and hyperparameters.
+#
+# Parameters in this file are set accoring to the training of the Icelandic g2p models published in
```
Comment (lumpidu) [inline sweep-is]: typo

File: README.md [hunk 2]
```diff
@@ -34,6 +33,32 @@ conda activate fairseq-lstm
 
 2. Clone Fairseq and install, see: https://github.com/pytorch/fairseq
 
+## Training and executing
+
+This repository contains trained models which you can use to automatically transcribe Icelandic words.
```
Comment (lumpidu) [inline README.md]: Could you add a relative mardown link to the directory of the models ? E.g. [models](checkpoints/)

Comment (bnika) [reply README.md]: fixed, and also corrected the links for issues and pull requests to be the icelandic-lt ones instead of grammatek

**Comment by lumpidu** [comment]
LGTM

**Pull request merged**